stacks/python-django/devfile.yaml

```diff
@@ -1,20 +1,19 @@
 schemaVersion: 2.1.0
 metadata:
   name: python-django
-  version: 2.1.0
   displayName: Django
-  description:
-    'Django is a high-level Python web framework that enables rapid development of secure and maintainable websites.
+  description: "Django (version 3.x) is a high-level Python web framework that enables rapid development of secure and maintainable websites.
```

Comment:
django version is not defined in the image or devfile, but it will depend on the application dependencies